src/testGoogle.cpp

```diff
@@ -0,0 +1,88 @@
+#include "gtest/gtest.h"
+#include "ecfeed.hpp"
+#include <iostream>
+
+// conan install src/conanfile.txt -g compiler_args
+// clear && g++ src/testGoogle.cpp @conanbuildinfo.args -o test -std=c++17 && ./test
```

Comment:
how does it link to an executable without a main function?

Reply:
In case of missing 'main', Google Test provides its own implementation.